test/integration/resultsPageErrors.js

```diff
@@ -0,0 +1,61 @@
+const cheerio = require('cheerio');
```

Comment:
Can all external dependencies be listed before internal modules please? Ideally alphabetically.